doc/api/test.md

```diff
@@ -219,6 +219,67 @@ test('a test that creates asynchronous activity', (t) => {
 });
 ```
 
+## Running tests from the command line
+
+The Node.js test runner can be invoked from the command line by passing the
+[`--test`][] flag:
+
+```bash
+node --test
+```
+
+By default, Node.js will recursively search the current directory for
+JavaScript source files matching a specific naming convention. Matching files
```

Comment:
Nit: What is the specific naming convention? Might be good to state that here up front.

Reply:
Its linked in this paragraph.

Reply:
Ha! I guess I haven't had nearly enough coffee this morning or I got entirely too much sun at the beach the past two days... my eyes missed that link entirely!!